cockatrice/src/player.cpp

```diff
@@ -2641,12 +2738,22 @@ void Player::actDecPT()
     actIncPT(-1, -1);
 }
 
+void Player::actFlowP()
+{
+    actIncPT(1, -1);
+}
+
+void Player::actFlowT()
+{
+    actIncPT(-1, 1);
+}
+
 void Player::actSetAnnotation()
 {
     QString oldAnnotation;
-    QListIterator<QGraphicsItem *> i(scene()->selectedItems());
-    while (i.hasNext()) {
-        auto *card = static_cast<CardItem *>(i.next());
+    auto sel = scene()->selectedItems();
+    for (const auto &item : sel) {
```

Comment:
can simplify further
```suggestion
    for (const auto &item : scene()->selectedItems()) {
```